drivers/pds/lib/utils.go

```diff
@@ -1886,6 +1886,7 @@ func RunMySqlWorkload(dnsEndpoint string, pdsPassword string, namespace string,
 	value = append(value, dnsEndpoint)
 	value = append(value, "pds")
 	value = append(value, pdsPassword)
+	value = append(value, pdsPort)
```

Comment:
Can we have a function to get the port dynamically from connection-info API?

Reply:
Created a separate PA Ticket for this.